lizmap/modules/lizmap/controllers/edition.classic.php

```diff
@@ -961,8 +961,15 @@ public function saveFormDataToDb($form){
               $value = 'NULL';
             break;
           case 'text':
+            if ( !$value or strlen(strtolower($value)) == 0)
+              $value = 'NULL';
+            else
+              $value = $cnx->quote($value);
           case 'boolean':
-            if ( !$value or empty($value))
+            if(strtolower($value) != 'true' &&  $value !== 't' &&
```

Comment:
if $value == 'T', it won't pass the test. Probably you want to do strtolower before any tests...

Reply:
yes, I will add it for boolean.